src/cpdb/helpers.py

```diff
@@ -16,14 +41,44 @@ def get_data(branch) -> dict:
         aws_secret_access_key = os.getenv("AWS_SECRET_ACCESS_KEY"),
         endpoint_url = os.getenv("AWS_S3_ENDPOINT")
     )
-    for table in ['magency', 'sagency']:
-        obj = client.get_object(
-            Bucket="edm-private",
-            Key=f"db-cpdb/{branch}/latest/output/analysis/cpdb_summarystats_{table}.csv",
-        )
-        s = str(obj["Body"].read(), "utf8")
-        data = StringIO(s)
-        df = pd.read_csv(data, encoding='utf8')
-        rv[table] = df
-
-    return rv
\ No newline at end of file
+    #for table in ['magency', 'sagency']:
+    obj = client.get_object(
```

Comment:
I think this is tough to refactor now because the ingestion is totally adhoc because of the where the previous version lives currently (not on open data). I think once open data discussion is further along we can have a more solid conversation about how to refactor this ingestion process which probably looks very different from its current state.

Reply:
that's fair - i think abstracting the data ingestion is good because it facilitates more easily changing it in the future, but dont think its a blocker or anything

Reply:
Sorry Te can you write more here about this issue with the data ingestion? I'm looking at the [upload step in cpdb](https://github.com/NYCPlanning/db-cpdb/blob/1009c9105ff26af8d68359be63f346197efcafb2/cpdb.sh#L43) and it seems like we should be able to consistently get the most recent version of cpdb from that `db-cpdb/{branch}/latest/output/analysis/cpdb_summarystats_{table}.csv` path? What am I missing

Reply:
for the latest then yes. Question is about the last version before the previous and it is difficult to nail down which is the last "official" version before the current latest build if that makes sense. @SashaWeinstein